Sources/Basics/AuthorizationProvider.swift

```diff
@@ -71,6 +71,9 @@ extension AuthorizationProvider {
         guard let (user, password) = self.authentication(for: url) else {
             return nil
         }
+        guard user != "token" else {
```

Comment:
Using `token` as username for bearer auth is specific to registry auth. Adding this change would mean applying that logic for all SwiftPM auth. Is this what we want @MaxDesiatov?

Reply:
cc @bnbarham. I feel like we need to revert it if you think the behavior change (even though for making it consistent) is undesired.